website/docs/BasicInterfaces/Agent.md

```diff
@@ -1,58 +1,45 @@
 
 ```js
-const SecretAgent = require('secret-agent');
+const agent = require('secret-agent');
 
 (async () => {
-  const agent = new SecretAgent();
   await agent.goto('https://www.google.com');
   // other actions...
   await agent.close();
 })();
 ```
 
-A SecretAgent instance can be thought of as a single user browsing session. An instance has a [replayable](../advanced/session-replay)&nbsp;[Session](../advanced/session) that will record all commands, dom changes, interaction and page events. 
+An Agent instance can be thought of as a single user-browsing session. An instance has a [replayable](../advanced/session-replay)&nbsp;[Session](../advanced/session) that will record all commands, dom changes, interaction and page events.
 
-Instances are very lightweight, sharing a pool of browsers underneath. You should create a new instance for activity that would benefit from parallelization by multiple "users".
+A default instance is automatically initialized and available as the default export.
 
-SecretAgent instances can have multiple [Tabs](./tab), but only a single tab can be focused at a time. Clicks and other user interaction will go to the active tab (interacting with multiple tabs at once by a single user is easily detectable). 
-
-Each SecretAgent instance creates a private environment with its own cache, cookies, session data and [BrowserEmulator](../advanced/browser-emulators). No data is shared between instances -- each operates within an airtight sandbox to ensure no identities leak across requests.
-
-## Constructor
-
-### new SecretAgent*(options)* {#constructor}
-
-Creates a new sandboxed browser instance with [unique user session and fingerprints](../overview/basic-concepts). Or pass in an existing UserProfile.
-
-You can optionally await an instance (or constructor) to cause the connection to the underlying SecretAgent to be initialized. If you don't await, the connection will be established on the first call.
-
-Note: If you provide a `name` that has already been used to name another instance then a counter will be appended to your string to ensure its uniqueness. However, it's only unique within a single NodeJs process (i.e., rerunning your script will reset the counter).
+Instances are very lightweight, sharing a pool of browsers underneath. To generate multiple "scrapes" in a single script, you'll want to create a [Handler](./handler).
 
 ```js
-const SecretAgent = require('secret-agent');
+const { Handler } = require('secret-agent');
 
 (async () => {
-  // connection established here
-  const agent = await new SecretAgent();
+  const handler = new Handler();
+  handler.dispatchAgent(async agent => {
+    await agent.goto('https://www.google.com/search?q=secret-agent');
+  });
+  handler.dispatchAgent(async agent => {
+    await agent.goto('https://www.bing.com/search?q=secret+agent');
+  });
+  await handler.waitForAllDispatches();
 })();
```

Comment:
I'm not sure we want to give a Handler example... linking to Handler is probably enough. Diving into an example that's not relevant to the topic at hand makes the page too complex.

Reply:
I started this off with a mention of Handler, and it felt like I was not doing justice to explaining what all you can do. I think your thought is very valid though. An "Agent" comes from 2 places - a handler, or a default export. This "file" explains how to use it, so it feels very relevant to explain how you get one. Since this "class" is no longer "constructed", it seemed like we need to be showing you how that happens.